app/_redirects

```diff
@@ -671,12 +675,12 @@
 
 # GENERAL REDIRECTS TO /GATEWAY/
 /enterprise/changelog      /gateway/changelog
-/enterprise/2.6.x/introduction/key-concepts     /konnect-platform/key-concepts
-/enterprise/2.7.x/introduction/key-concepts     /konnect-platform/key-concepts
-/enterprise/latest/introduction/key-concepts    /konnect-platform/key-concepts
-/enterprise/2.6.x/introduction/support-policy   /konnect-platform/support-policy
-/enterprise/2.7.x/introduction/support-policy   /konnect-platform/support-policy
-/enterprise/latest/introduction/support-policy  /konnect-platform/support-policy
+/enterprise/2.6.x/introduction/key-concepts     /gateway/latest/availability-stages/
+/enterprise/2.7.x/introduction/key-concepts     /gateway/latest/availability-stages/
+/enterprise/latest/introduction/key-concepts    /gateway/latest/availability-stages/
+/enterprise/2.6.x/introduction/support-policy   /gateway/latest/support-policy/
+/enterprise/2.7.x/introduction/support-policy   /gateway/latest/support-policy/
+/enterprise/latest/introduction/support-policy  /gateway/latest/support-policy/
```

Comment:
`/gateway/latest/support-policy/` is resulting in a 404, even when you click it in the sidebar nav